### PR TITLE
[NO-JIRA] Add accessibility guideline pages

### PR DESCRIPTION
**Pull request by georgegillams**
This is WIP, so we're not adding it to the side-nav yet, but we'd like to get it live to:
a) allow parallelisation
b) start testing it

image

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/pages/AccessibilityDesignPage/DesignColorAlone.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const DesignColorAlone = () => (
+  <Fragment>
+    <BpkMarkdownRenderer source={getMarkdownString(colorAlone1)} />
+    <div
+      className={getClassName(
+        'bpk-docs-accessibility-design-page__do-dont-wrapper',
+      )}
+    >
+      <BpkDoDont
```
Comment (georgegillams) [inline docs/src/pages/AccessibilityDesignPage/DesignColorAlone.js]: These examples need replacing with the correct images

File: .spelling [hunk 2]
```diff
@@ -98,6 +98,7 @@ Artifactory
 Cocoapods
 CommonJS
 ESLint
+Figma
```
Comment (olliecurtis) [inline .spelling]: 😱

File: docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js [hunk 3]
```diff
@@ -40,8 +40,8 @@ import STYLES from './BpkMarkdownRenderer.scss';
 
 const getClassName = cssModules(STYLES);
 
-const TAG_NAMES = ['h1', 'h1', 'h2', 'h3'];
-const TEXT_STYLES = ['xxl', 'xxl', 'xl', 'lg'];
+const TAG_NAMES = ['h1', 'h1', 'h2', 'h3', 'h4'];
+const TEXT_STYLES = ['xxl', 'xxl', 'xl', 'lg', 'base'];
```
Comment (olliecurtis) [inline docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js]: I know not part of this but should we add our other supported sizes to this list? `md`, `sm`, `xs`?

File: docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js [hunk 4]
```diff
@@ -101,6 +101,8 @@ const BpkMarkdownRenderer = (props: Props) => {
 
   renderers.heading = headingProps => {
     const { level, children, ...headingRest } = headingProps;
+    console.log(`level`, level);
```
Comment (olliecurtis) [inline docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js]: Did you mean to leave this in?

File: docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js (hunk 3 above)
Comment (georgegillams) [reply docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js]: Well the heading level should only be 1-4

Comment (olliecurtis) [reply docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js]: Ok thats fine, just curious mainly as our `BpkText` component accepts different text styles but that might be another discussion 😄

**Pull request merged**